packages/styles/src/variables/components/_notification.scss

```diff
@@ -70,20 +70,14 @@ $notification-icon-size-map: map.merge(
 );
 
 $notification-variants: () !default;
-$notification-variants: list.join(
-  $notification-variants,
-  (
-    'primary' color.$gray-80 3134,
-    'success' color.$success-green 2105,
-    'danger' color.$error 2104,
-    'warning' color.$warning 2104,
-    'info' color.$info 2106,
-    'gray' color.$gray-background 2201,
-    // deprecated
-    'notification' color.$gray-80 3134,
-    'error' color.$error 2104
-  )
-);
+$notification-icons: (3134, 2105, 2104, 2104, 2106, 2201, 3134, 2104);
```

Comment:
A list makes it difficult to know which color corresponds to which icon.

Reply:
Update with our discussion's output